src/transformers/models/altclip/modeling_altclip.py

```diff
@@ -1217,6 +1224,9 @@ def __init__(self, config, add_pooling_layer=True):
 
         self.pooler = AltRobertaPooler(config) if add_pooling_layer else None
 
+        self._use_sdpa = config._attn_implementation == "sdpa"
+        self.position_embedding_type = config.position_embedding_type
```

Comment:
It's a bit funny here setting `self._use_sdpa` as the model doesn't have an SDPA attention class to use. 

I think it's just about passable, as the param is private and an error will be thrown earlier when we try to instantiate the attention layer.